aws/resource_aws_spot_fleet_request_test.go

```diff
@@ -937,24 +1248,35 @@ resource "aws_iam_role" "test-role" {
 EOF
 }
 
+resource "aws_launch_template" "foo" {
+  name = "test-launch-template"
+  image_id = "ami-516b9131"
+  instance_type = "m1.small"
+  key_name = "${aws_key_pair.debugging.key_name}"
+}
+
 resource "aws_spot_fleet_request" "foo" {
     iam_fleet_role = "${aws_iam_role.test-role.arn}"
     spot_price = "0.005"
     target_capacity = 2
     valid_until = "2019-11-04T20:44:20Z"
-    fleet_type = "request"
     terminate_instances_with_expiration = true
+    instance_interruption_behaviour = "stop"
     wait_for_fulfillment = true
-    launch_specification {
-        instance_type = "m1.small"
-        ami = "ami-516b9131"
+
+    launch_template_configs {
+      launch_template_specification {
+        name = "${aws_launch_template.foo.name}"
```

Comment:
I did try with ids but because they are generated each time I found it impossible to predict the resource hash required to check the results; any ideas?